git_machete/git_operations.py

```diff
@@ -165,6 +165,11 @@ class BranchPair(NamedTuple):
     local_or_remote_branch: AnyBranchName
 
 
+class OrganizationRepository(NamedTuple):
```

Comment:
This is GitHub-specific and should probably live in github.py. Vanilla git doesn't know anything about _organizations_.